pontoon/base/models.py

```diff
@@ -1800,20 +1800,21 @@ def calculate_stats(self, save=True):
 
         # Plural
         nplurals = locale.nplurals or 1
-        partial = 0
+        missing = 0
         for e in translated_entities.exclude(string_plural=''):
             translations = Translation.objects.filter(entity=e, locale=locale)
             plural_approved_count = translations.filter(approved=True).count()
             plural_fuzzy_count = translations.filter(fuzzy=True).count()
+            plural_translated_count = translations.values('plural_form').distinct().count()
 
             if plural_approved_count == nplurals:
                 approved += 1
             elif plural_fuzzy_count == nplurals:
                 fuzzy += 1
-            elif plural_approved_count > 0 or plural_fuzzy_count > 0:
-                partial += 1
+            elif plural_translated_count < nplurals:
```

Comment:
I remember that at some point this condition looked like this. Anyway, this looks a lot more reasonable now.